tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -10,7 +10,6 @@
 import pytest
 
 import host_tools.drive as drive_tools
-from framework.artifacts import NetIfaceConfig
```

Comment:
`NetIfaceConfig` is unimported but is still used in "[test: clean up network interfaces and SSH helper](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/a8fb4b7369f8ede378399a88cb42adee05b25990)"

Reply:
fixed on my forked repo by using the returned value of `vm.add_net_iface()` temporarily and removed it in "[test: revamp snapshot support to Microvm](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/3197cde20ebb18fe9b6f96a66f43432d9760c06b)": https://github.com/zulinx86/firecracker/commits/new-ci-artifacts
- https://github.com/zulinx86/firecracker/commit/16d33556238c5cc1ea94bd08afe6a80649fbc5e4#diff-d960bea365831acfb0eb3b1b548e6d22293710c9ed558f5dfbf68e016457870dR66
- https://github.com/zulinx86/firecracker/commit/772aa1fb8672137407258173f7cb72b3e2af0bdc#diff-d960bea365831acfb0eb3b1b548e6d22293710c9ed558f5dfbf68e016457870dL66-R65